cost/aws/shared_cost_reallocation/aws_support/shared_cost_reallocation_aws_support.pt

```diff
@@ -0,0 +1,1124 @@
+name "Shared Cost Reallocation - AWS Support"
```

Comment:
Policy template should be added to the list of validated policy templates (https://github.com/flexera-public/policy_templates/blob/master/tools/policy_master_permission_generation/validated_policy_templates.yaml)